test/test_user_interface.py

```diff
@@ -1,10 +1,41 @@
 import unittest
 from unittest.mock import patch
 
+from src.message import Message
 from src.user_interface import UserInterface
 
 
 class TestUserInterface(unittest.TestCase):
+    @patch("builtins.print")
+    def test_display_message_prints_welcome_message_passed_in(self, mock_print):
+        user_interface = UserInterface()
+        message = Message()
+        message = message.welcome_message()
+        user_interface.display_message(message)
+        mock_print.assert_called_with("Welcome to Tic Tac Toe")
```

Comment:
What are we actually testing here?  Just that a message passed in gets displayed?  Can we do this without having to use the `Message` class at all?

Reply:
Okay I removed the call to `Message()` and just test that it is printing a string I send it to it.

Reply:
this looks good.

Reply:
now can we remove the dependency on `Message`? by deleting this line:
`from src.message import Message`